src/components/Checkbox.js

```diff
@@ -11,20 +11,29 @@ const propTypes = {
 
     /** A function that is called when the box/label is pressed */
     onPress: PropTypes.func.isRequired,
+
+    /** Should the input be styled for errors  */
+    hasError: PropTypes.bool,
+};
+
+const defaultProps = {
+    hasError: false,
 };
 
 const Checkbox = ({
     isChecked,
     onPress,
+    hasError,
 }) => (
     <Pressable onPress={() => onPress(!isChecked)}>
-        <View style={[styles.checkboxContainer, isChecked && styles.checkedContainer]}>
+        <View style={[styles.checkboxContainer, isChecked && styles.checkedContainer, hasError && styles.borderColorDanger]}>
```

Comment:
Can you move the style prop to its own line and put each style on a new line? There's an example in the `// Good` section of https://github.com/Expensify/App/blob/main/STYLING.md#when-to-create-a-new-style